web-console/src/views/servers-view.tsx

```diff
@@ -100,14 +123,37 @@ export class ServersView extends React.Component<ServersViewProps, ServersViewSt
     );
   }
 
+  static getServers = async (): Promise<ServerQueryResultRow[]> => {
+    const allServerResp = await axios.get('/druid/coordinator/v1/servers?simple');
+    const allServers = allServerResp.data;
+    return allServers.filter((s: any) => s.type === 'historical').map((s: any) => {
+      return {
+        host: s.host.split(':')[0],
+        plaintext_port: parseInt(s.host.split(':')[1], 10),
+        server: s.host,
+        curr_size: s.currSize,
+        max_size: s.maxSize,
+        tier: s.tier,
+        tls_port: -1
+      };
+    });
+  }
+
   componentDidMount(): void {
+    const { noSqlMode } = this.props;
     this.serverQueryManager = new QueryManager({
       processQuery: async (query: string) => {
-        const servers = await queryDruidSql({ query });
-
+        let servers: ServerQueryResultRow[];
+        if (!noSqlMode) {
+          servers = await queryDruidSql({ query });
+          if (servers.length === 0) {
```

Comment:
could you add a comment here about why we are doing this, also link in the comment to https://github.com/apache/incubator-druid/issues/7498 and https://github.com/apache/incubator-druid/issues/7005

Reply:
Will implement another way of querying the servers in a separate PR